go/src/kafka-pubsub-emulator-gateway/build.sh

```diff
@@ -28,7 +28,7 @@ clean() {
 }
 
 build_proto_files(){
-  go get -u -v github.com/spf13/cobra/cobra && \
+  go get -u -v github.com/spf13/cobra && \
```

Comment:
Shouldn't this be 

```
go get -u -v gthub.com/spf13/cobra-cli
```

?

Reply:
I don't think so. From what I can tell, this is trying to fetch the dependencies needed to compile, and that is only the cobra lib. It just happened to fetch the parent lib as a side effect before

Reply:
> this is trying to fetch the dependencies needed to compile

Ah that makes sense 👍🏼